### PR TITLE
test: add test for field ordering in triggers

### PR DESCRIPTION
**Pull request by quisquous**
Functions run in a particular order, so enforce that order in how
triggers are written to avoid logic errors.

### CODE REVIEW

**Comment by quisquous** [comment]
Did anybody have any feelings about this? I know there was some confusion about when things run, and so having fields ordered by when they run seems like it would only help in trigger readability, even if it makes Travis be even more of a jerk.

(cc @panicstevenson)

**Comment by panicstevenson** [comment]
It seems good to me; maybe I'll finally remember to run tests 100% of the time 🤞

File: ui/raidboss/popup-text.js [hunk 1]
```diff
@@ -587,11 +588,8 @@ class PopupText {
         }
       }
 
-      if ('soundVolume' in trigger)
-        soundVol = ValueOrFunction(trigger.soundVolume);
-
       soundUrl = triggerOptions.SoundOverride || soundUrl;
-      soundVol = triggerOptions.VolumeOverride || soundVol;
+      soundVol = triggerOptions.VolumeOverride || triggerSoundVol || soundVol;
```
Comment (panicstevenson) [inline ui/raidboss/popup-text.js]: Is this from a separate change?

Comment (quisquous) [reply ui/raidboss/popup-text.js]: Reply for others, it's from the middle commit d5b001c.

**Pull request merged**